boards/common/nrf52/Makefile.features

```diff
@@ -3,7 +3,7 @@ CPU = nrf52
 # Put defined MCU peripherals here (in alphabetical order)
 FEATURES_PROVIDED += periph_rtt
 FEATURES_PROVIDED += periph_timer
-ifeq (,$(filter nordic_softdevice_ble,$(USEPKG)))
+ifeq (,$(filter nordic_softdevice_ble,$(USEPKG))$(filter bootloader_nrfutil,$(FEATURES_PROVIDED)))
```

Comment:
I don't think it's necessary to change that file.

Reply:
See discussion at https://github.com/RIOT-OS/RIOT/pull/13146#discussion_r421676370 -- the alternative seems to duplicate that file except its riotboot component. (The exact execution of the new filter expression is topic of the very latest main-thread discussion.)

Reply:
riotboot is automatically blacklisted when the bootloader_nrfutil feature is required. So add `FEATURES_REQUIRED += bootloader_nrfutil` in Makefile.dep like I suggest below and you don't have to change this file.

Reply:
> riotboot is automatically blacklisted when the bootloader_nrfutil feature is required

Sorry, that's not true. But there's no issue raised by the build system if both are used at the same time, although I doubt that will work.

Reply:
@benpicco, you initially raised the issue that riotboot needs to be removed. Why is that necessary?

Reply:
> But there's no issue raised by the build system if both are used at the same time

Actually there is - that's the reason for this whole dance

Reply:
So why wasn't this necessary for adafruit-clue, which is in the same case ?

Reply:
That seems to be the crucial question to me; I'll look into the upcoming push's errors to understand that.